hydrator-common/src/main/java/co/cask/hydrator/common/SourceInputFormatProvider.java

```diff
@@ -38,6 +40,12 @@ public SourceInputFormatProvider(Class<? extends InputFormat> inputFormatClass,
 
   public SourceInputFormatProvider(String inputFormatClassName, Configuration hConf) {
     Map<String, String> config = new HashMap<>();
+    // Unset the values of the default filesystem and implementation of the filesystem
```

Comment:
Maybe we should have a method in JobUtils:
`public Map<String, String> getConfiguration(Job job);`
so that the unsetting logic is in the same class as the setting of these parameters.